packages/generator-nitro/generators/app/templates/package.json

```diff
@@ -82,8 +82,7 @@
     "lazysizes": "5.1.2",
     "lodash": "4.17.15",
     "lottie-web": "5.5.9",
-    "prevent-window-opener-attacks": "0.2.4",<% } %><% if (options.jsCompiler === 'ts') { %>
-    "promise-polyfill": "8.1.3",<% } %>
+    "prevent-window-opener-attacks": "0.2.4",<% } %>
     "regenerator-runtime": "0.13.3"<% if (options.exampleCode) { %>,
```

Comment:
The "regenerator-runtime" is not included in the typescript project. 
Without automated polyfilling we probably don't need this in a typescript project.